flink-runtime/src/main/java/org/apache/flink/runtime/operators/coordination/OperatorCoordinatorHolder.java

```diff
@@ -401,6 +409,10 @@ private void checkInitialized() {
 			checkState(isInitialized(), "Context was not yet initialized");
 		}
 
+		private void resetFailed() {
```

Comment:
Nit: The other methods called from Holder class are all "package private" to avoid bridge methods.